V4_MIGRATION_GUIDE.md

```
@@ -0,0 +1,491 @@
+- [**Breaking Changes**](#breaking-changes)
+  + [Classes Removed](#classes-removed)
+  + [Deprecated MFA Methods Removed from AuthenticationAPIClient](#deprecated-mfa-methods-removed-from-authenticationapiclient)
+  + [DPoP Configuration Moved to Builder](#dpop-configuration-moved-to-builder)
+  + [DPoPException.UNSUPPORTED_ERROR Removed](#dpopexceptionunsupported_error-removed)
+  + [SSOCredentials.expiresIn Renamed to expiresAt](#ssocredentialsexpiresin-renamed-to-expiresat)
+  + [SecureCredentialsManager Auth0 Constructors Removed](#securecredentialsmanager-auth0-constructors-removed)
+- [**Default Values Changed**](#default-values-changed)
+  + [Credentials Manager minTTL](#credentials-manager-minttl)
+- [**Behavior Changes**](#behavior-changes)
+  + [CredentialsManager Now Uses the Global Executor](#credentialsmanager-now-uses-the-global-executor)
+  + [clearCredentials() Now Clears All Storage](#clearCredentials-now-clears-all-storage)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid
Expected: `#clearcredentials-now-clears-all-storage`; Actual: `#clearCredentials-now-clears-all-storage`

(MD051, link-fragments)